SuperBuild/External_Elastix.cmake

```diff
@@ -7,7 +7,7 @@ set(${proj}_DEPENDENCIES "ITK")
 file( WRITE "${CMAKE_CURRENT_BINARY_DIR}/${proj}-build/CMakeCacheInit.txt" "${ep_common_cache}" )
 
 set( ELASTIX_GIT_REPOSITORY ${git_protocol}://github.com/SuperElastix/elastix )
-set( ELASTIX_GIT_TAG d44e65e119f0f6b22cfaccd03bc5269484e9c137 ) # hash July 10, 2024
+set( ELASTIX_GIT_TAG 5.2.0 )
```

Comment:
For your information, these "tags" are just six commits apart:
https://github.com/SuperElastix/elastix/compare/d44e65e119f0f6b22cfaccd03bc5269484e9c137...5.2.0

However, one of those six commits is a significant bug fix:
- Pull request https://github.com/SuperElastix/elastix/pull/1195 commit https://github.com/SuperElastix/elastix/commit/e3cafc5c1cd5d1eda4736984c3d3050d9353cc6f

So I think it's worth it  😃